webrender/src/glyph_cache.rs

```diff
@@ -89,6 +90,14 @@ impl GlyphCache {
         }
     }
 
+    pub fn clear(&mut self, texture_cache: &mut TextureCache) {
+        let caches = mem::replace(&mut self.glyph_key_caches, FastHashMap::default());
```

Comment:
> Bonus points - space is kept reserved.

Well, this is the one situation where keeping the space reserved is undesirable. But I can just do the loop as you suggest and assign an empty hash map to it instead of clearing if you prefer.